soc/arc/snps_nsim/CMakeLists.txt

```diff
@@ -23,6 +23,17 @@ else()
 			       -Xfpus_div -Xfpu_mac -Xfpuda -Xfpus_mpy_slow
 			       -Xfpus_div_slow -Xbitstream -Xtimer0 -Xtimer1)
 
+  zephyr_compile_options_ifdef(CONFIG_SOC_NSIM_EM11D -arcv2em -core3 -Xdiv_rem=radix2
+			       -Xmpy_option=mpyd -Xbitscan -Xswap -Xbarrel_shifter
+			       -Xshift_assist -Xdsp2 -Xdsp_complex
+			       -Xdsp_divsqrt=radix2 -Xdsp_itu -Xdsp_accshift=full
```

Comment:
Hmm, do we sure that instructions generated after we pass `-Xdsp2` `-Xdsp_complex` `-Xdsp_itu` `-Xdsp_accshift=full` there will be no instructions generated which can change any value except 64-bit accumulator (which we do save/restore every time).

Reply:
Also move `-Xmpy_option=mpyd` to dsp options, as it need `-Xdsp2`